### PR TITLE
fix: Attempt at grouping lockfile maintenance

### PR DESCRIPTION
**Pull request by another-rex**
Not sure if this will work, but it will be quite nice if all the lockfile maintenance can be done in 1 PR to reduce weekly chore.

### CODE REVIEW

**Review by hogo6002 — APPROVED** [review]

**Comment by andrewpollock** [comment]
What was the reasoning behind the groupings in the first place?

**Review by andrewpollock — APPROVED** [review]

**Comment by another-rex** [comment]
Grouping was for updating specific packages, it makes sense to review one project at a time (so you can approve the docs update, but change/fix/hold off the worker dependency update.).

With lockfile maintenance, the workflow (at least for me) is literally just approving it as long as everything looks roughly fine and only the lockfile is changed. There's no need for this to be separate. (Apparently the default is supposed to be just 1 group? not sure how it got split up)

**Pull request merged**
